jackal_control/launch/control.launch.py

```diff
@@ -55,23 +55,23 @@ def generate_launch_description():
     # Localization
     localization_group_action = GroupAction([
         # Extended Kalman Filter
-        Node(
-            package='robot_localization',
-            executable='ekf_node',
-            name='ekf_node',
-            output='screen',
-            parameters=[config_jackal_ekf],
-        ),
+#        Node(
```

Comment:
This needs to be enabled.

Reply:
I had this disabled as I was testing my real robot using a different odometry package for 2D laser scans - [rf2o_laser_odometry](https://github.com/MAPIRlab/rf2o_laser_odometry). I'll enable this again for now.